lib/iris/coord_systems.py

```diff
@@ -820,3 +820,51 @@ def as_cartopy_crs(self):
 
     def as_cartopy_projection(self):
         return self.as_cartopy_crs()
+
+
+class Mercator(CoordSystem):
+    """
+    A coordinate system in the Mercator projection.
+
+    """
+
+    grid_mapping_name = "mercator"
+
+    def __init__(self, central_lon=0, ellipsoid=None):
+        """
+        Constructs a Mercator coord system.
+
+        Args:
+            * central_lon
+                    The central longitude.
+
+        Kwargs:
+
```

Comment:
I think we normally have a blank line here, the `Args:` section should have one too.